crawl-ref/source/mon-death.cc

```diff
@@ -2028,13 +2029,15 @@ item_def* monster_die(monster& mons, killer_type killer,
 
     // Adjust song of slaying bonus. Kills by relevant avatars are adjusted by
     // now to KILL_YOU and are counted.
-    if (you.duration[DUR_SONG_OF_SLAYING]
+    if (you.duration[DUR_WEREBLOOD]
```

Comment:
weird place for an effect like this, it would be better if this were somehow done not as a side effect. (E.g. just keep track of deaths in a prop and do the howling in player_reacts.)

Reply:
(Or, I get the flavor of this, but it might be simpler for the player if it just had old SoS behavior and always made noise.)

Reply:
An earlier drat had the constant noise retained. The constant noise made the spell weaker, and imo too weak. 

In terms of the howl being in `monster_die` as a side effect, that function is where we give out XP, heal on kills, &c. so it's the correct place for an on-death side effect.

Reply:
I mean, all side effects are bad (and death side effects are a constant source of bugs) so I'm not in favor of adding more, and both of the ones you mention lead to issues too. In fact the xp one combine with skill targets is in major need of (difficult) refactoring because of the crash on disconnect bug.

Reply:
Hm. I think I'll go back to the old version but make the noise happen on a slay-bonus-in-max chance in `player_reacts`.